### PR TITLE
PORTALS-2449: Add Challenge type generic card (with associated icon) …

### PR DESCRIPTION
**Pull request by jay-hodgson**
…using the old ways, a generic Tabs portals component (that takes SynapseConfigs as input), and the new cards list for Active and Closed Challenges in the Challenge portal

### CODE REVIEW

**Comment by jay-hodgson** [comment]
Screen Shot 2023-05-15 at 5 52 25 PM
This is linked to the existing placeholder Challenge Detail page.

File: apps/portals/src/portal-components/TabbedSynapseObjects.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  return <Box className="TabbedSynapseObjects">
+    <Tabs
+      value={selectedTabConfig.label}
+      centered={centerTabs}
+    >
+      {tabConfigs.map((tabConfig, index) => {
+        return <Tab key={index} value={tabConfig.label} disableRipple={true}
+        label={tabConfig.label}
+        disableTouchRipple
+        onClick={() => setSelectedTabConfig(tabConfig)}/>
```
Comment (nickgros) [inline apps/portals/src/portal-components/TabbedSynapseObjects.tsx:28]: `disableRipple` and `disableTouchRipple` could be added as `defaultProps` in the theme if we never want them

File: apps/portals/src/portal-components/TabbedSynapseObjects.tsx
Comment (nickgros) [inline apps/portals/src/portal-components/TabbedSynapseObjects.tsx:1]: Hopefully these tabs don't need custom routing! Is it likely we'll have multiple tab groups per page?

**Review by nickgros — APPROVED** [review]
Looks great!

File: apps/portals/src/portal-components/TabbedSynapseObjects.tsx
Comment (jay-hodgson) [reply apps/portals/src/portal-components/TabbedSynapseObjects.tsx:1]: That's a good point.  I don't think we need to restore the tab selection state in this case, but in the future we could add an optional variable that could be utilized (to store in a search parameter if sharing is important, or local storage).

**Pull request merged**